### PR TITLE
Numberbox: pressing the "." key on the numpad should clear the selected text regardless of the char produced (T1199553)

### PR DESCRIPTION
**Pull request by iBat**

### CODE REVIEW

File: packages/devextreme/js/ui/number_box/number_box.mask.js [hunk 1]
```diff
@@ -187,7 +187,9 @@ const NumberBoxMask = NumberBoxBase.inherit({
         const normalizedText = this._getInputVal();
         const caret = this._caret();
 
-        const enteredChar = this._lastKeyName === MINUS_KEY ? '' : this._lastKey;
+        const enteredChar = this._lastKeyName === MINUS_KEY
+            ? ''
+            : e.which === 110 ? number.getDecimalSeparator() : this._lastKey;
```
Comment (marker-dao) [inline packages/devextreme/js/ui/number_box/number_box.mask.js]: What's 110? Let's use a variable with a descriptive name.

Comment (alexanderPolosatov) [inline packages/devextreme/js/ui/number_box/number_box.mask.js]: let move this internal condition to variable or refactor to 'if'

Comment (marker-dao) [inline packages/devextreme/js/ui/number_box/number_box.mask.js]: Let's simplify the use of the ternary operator.

File: packages/devextreme/testing/tests/DevExpress.ui.widgets.editors/numberbox.localization.tests.js [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+        this.keyboard
+            .caret({ start: 0, end: 6 })
+            .type('.');
+
+        assert.strictEqual(this.input.val(), '0.00', 'mask value is cleared');
+    });
+
+    QUnit.test('pressing the "." key on the numpad should clear the selected text regardless of the char produced (T1199553)', function(assert) {
+        const curLocale = localization.locale();
```
Comment (EugeniyKiyashko) [inline packages/devextreme/testing/tests/DevExpress.ui.widgets.editors/numberbox.localization.tests.js]: It's better not to abbreviate variable names. 
It is considered good practice to use clear and descriptive variable names.

File: packages/devextreme/testing/tests/DevExpress.ui.widgets.editors/numberbox.localization.tests.js [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+
+const moduleConfig = {
+    beforeEach: function() {
+        this.$element = $('#numberbox').dxNumberBox({
+            format: '#0.##',
+            value: '',
+            useMaskBehavior: true
+        });
+        this.clock = sinon.useFakeTimers();
+        this.input = this.$element.find('.dx-texteditor-input');
```
Comment (EugeniyKiyashko) [inline packages/devextreme/testing/tests/DevExpress.ui.widgets.editors/numberbox.localization.tests.js]: It's better to store class names at the top of the file inside constant variables.

**Review by EugeniyKiyashko — APPROVED** [review]

**Review by alexanderPolosatov — APPROVED** [review]

**Pull request merged**